### PR TITLE
Fix is_ground_content settings for nodes, remove line if the default of true

### PR DESCRIPTION
**Pull request by paramat**
Trees and leaves being false seems odd to me but i have left them that way as it seems intentional, does anyone know why? Trees are generated after caves but are still affected by large caves overgenerating from neighbouring mapchunks.
EDIT: Apparently it's to minimise cavegen griefing.

### CODE REVIEW

File: mods/beds/api.lua [hunk 1]
```diff
@@ -107,3 +109,4 @@ function beds.register_bed(name, def)
 		recipe = def.recipe
 	})
 end
+
```
Comment (rubenwardy) [inline mods/beds/api.lua]: There's two newlines here. You only need one. (Github hides the latter)

**Comment by Sokomine** [comment]
Maybe trees and leaves are considered ground_content so that when a neighbouring mapchunk creats a larger cave floating trees above removed ground are less likely to happen.
The change looks fine. Hope it gets merged so that more nodes are safe.

**Comment by paramat** [comment]
rubenwardy, looks like my text editor (gedit) removes newlines at EOF, so i end up adding another, other files in my commit have 2 newlines, this is confusing, i will remove the newlines i have added then.

**Comment by paramat** [comment]
Added newlines at end of files removed as they may already be there.

**Comment by sfan5** [comment]
:+1:

**Comment by paramat** [comment]
e15fde162457503d9cea45957784e4e66655a771

**Pull request closed without merging**